### PR TITLE
Fix Linting Error

### PR DESCRIPTION
**Pull request by billpull**

### CODE REVIEW

**Comment by chriskrycho** [comment]
Huh. Why in the world would `ember try:one` be invalid?

**Comment by kategengler** [comment]
`ember-try` was moved from a dependency of `ember-cli` to something in the project package.json with 3.0. The real question is how was it working before?

**Comment by chriskrycho** [comment]
@kategengler thank you for that clarification – makes sense. I must have missed that when I was updating us to latest Ember CLI.

**Comment by billpull** [comment]
Last issue seems to be the same as https://github.com/DockYard/ember-admin/issues/83

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "ember-try-placeholder",
+  "dependencies": {
```
Comment (kategengler) [inline bower.json]: This shouldn't be committed

**Comment by kategengler** [comment]
You removed `ember-source` from package.json, which is why the error about template compiler.

Bower is no longer used to test canary -- use a config that looks like: https://github.com/ember-cli/ember-addon-output/blob/master/config/ember-try.js

**Comment by billpull** [comment]
@kategengler thank you so much for the help

**Pull request merged**